packages/config/src/layer2s/index.ts

```diff
@@ -49,6 +51,7 @@ export const layer2s: Layer2[] = [
   nova,
   omgnetwork,
   optimism,
+  scroll,
```

Comment:
I'm not seeing this project even after checking the upcoming rollups checkbox. This is probably related to the API returning 0 tvl for it.

Reply:
yeap, I have addressed this issue in the recent commits